docs/user/collected-metrics/metrics.md

```diff
@@ -0,0 +1,63 @@
+<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
+
+# Metrics
+This document enumerates the metrics collected by this project.
```

Comment:
It would be nice if we could be more specific than "this project".  I would love for this to say "Glean itself".  But that's probably a separate bug for another time...

Reply:
We have [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1578347) already filed for making that better ;)